restcomm/restcomm.application/src/main/webapp/WEB-INF/scripts/mariadb/sql/conference-detail-records.xml

```diff
@@ -38,6 +38,33 @@
 		</if>
 		;
 	</select>
+        <!-- use artificial condition (1=1) that always evaluates to true to ease 
+        query composition. 
+        All parameters here are optional.
+        For "status" column use "like" operator to allow finding similar states.
+        -->
+        <select id="countByFilter" parameterType="org.restcomm.connect.dao.entities.ConferenceRecordCountFilter" resultType="int">
+		SELECT COUNT(*) FROM "restcomm_conference_detail_records" WHERE '1' = '1'
```

Comment:
can we recheck the syntax for mysql here, i think double quotes are not supported.

Reply:
i have just realized too Maria, good catch, i will change the sentence and test it in staging db as you suggested,thanks!!!